packages/itwin/grouping-mapping-widget/src/widget/components/GroupsView.tsx

```diff
@@ -51,8 +63,11 @@ export interface GroupsViewProps {
   setSelectedGroupForDeletion: (group: Group) => void;
   onDeleteGroup: (group: Group) => Promise<void>;
   onCloseDeleteModal: () => void;
-  numberOfVisualizedGroups?: number;
   alert?: React.ReactElement<typeof Alert>;
+  setIsOverlappedElementsInfoPanelOpen?: (isOverlappedElementsInfoPanelOpen: Group | undefined) => void;
+  isOverlappedElementsInfoPanelOpen?: Group | undefined;
```

Comment:
The name of this property suggests that it should be a boolean, however it is an optional Group.